provisor/utils.py

```diff
@@ -67,29 +67,22 @@ def validate_pubkey(value):
 
 
 def validate_username(value):
-    reserved_usernames = [
-        'about', 'abuse', 'main', 'data', 'example', 'jabber', 'legal', 'invite',
-        'copyright', 'contact', 'board', 'feedback', 'support', 'anonymous', 'index',
-        'inbox', 'payment', 'donate', 'calendar', 'dotfiles', 'billing', 'billings',
-        'images', 'media', 'policy', 'manage', 'messages', 'mobile', 'official',
-        'staging', 'development', 'staff', 'portal', 'forum', 'forums', 'pictures',
-        'photos', 'status', 'finger', 'private', 'press', 'user', 'users', 'username',
-        'usernames', 'sitemap', 'team', 'teams', 'account', 'accounts', 'chat', 'mail',
-        'email', 'admin', 'admins', 'administrator', 'administrators', 'postmaster',
-        'hostmaster', 'webmaster'
-    ]
+    from reserved import RESERVED_USERNAMES
 
     # Regexp must be kept in sync with
-    #  https://github.com/hashbang/hashbang.sh/blob/master/src/hashbang.sh#L178-191
+    #  https://github.com/hashbang/hashbang.sh/blob/master/src/hashbang.sh#L186-196
     if re.compile(r"^[a-z][a-z0-9]{,30}$").match(value) is None:
         raise ValueError('Username is invalid')
-    if value in reserved_usernames:
+
+    if value in RESERVED_USERNAMES:
         raise ValueError('Username is reserved')
+
     user_exists = True
     try:
         pwd.getpwnam(value)
     except:
         user_exists = False
+
```

Comment:
`except` what?
I know this isn't part of the PR, but it's staring me in the face.
Implicit Exception handling ain't cool.

Reply:
I have another branch which removes this code anyhow  ;)